test/test-imports/common.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
This script is intended to be `source`d so shebang is not required (and actually misleading)